trade.md

```diff
@@ -6,11 +6,14 @@ next:
 gcam-version: v5.1
 ---
 
-Commodities in GCAM are either traded globally (Heckscher-Ohlin) or fixed interregional trade.  Altough alternative approaches for trade can readily be implemented in the GCAM framework they are not currently implmented (with the exception of [GCAM USA](gcam-usa.html) where logit based decisions are made to facilitating trade between the 50-states).
+Commodities in GCAM are either traded globally (Heckscher-Ohlin, or Armington-ish) or fixed interregional trade.  Although alternative approaches for trade can readily be implemented in the GCAM framework they are not currently implemented (with the exception of [GCAM USA](gcam-usa.html) where logit based decisions are made to facilitating trade between the 50-states).
```

Comment:
International trade in most commodities in GCAM is done by one of three methods: (1) Heckscher-Ohlin or single global markets, (2) global trade with regionally-differentiated markets with Armington-like preferences between domestic and imported commodities, and (3) fixed interregional trade.  Other approaches for trade can also be implemented in the GCAM framework (such as [GCAM USA](gcam-usa.html) where logit based decisions are made to facilitating trade between the 50-states).